plugins/woocommerce/tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -21,17 +21,21 @@ wp-env run tests-cli "wp user create customer customer@woocommercecoree2etestsui
 echo -e 'Update Blog Name \n'
 wp-env run tests-cli "wp option update blogname 'WooCommerce Core E2E Test Suite'"
 
+echo -e 'Preparing Test Files \n'
+wp-env run tests-cli "sudo cp /var/www/html/wp-content/plugins/woocommerce/tests/legacy/unit-tests/importer/sample.csv \
+	/var/www/sample.csv"
+
 if [ $ENABLE_HPOS == 1 ]; then
-	echo 'Enable the COT feature'
+	echo -e 'Enable the COT feature\n'
```

Comment:
While we're at it ...

```suggestion
	echo -e 'Enable High-Performance Order Tables\n'
```